apps/web/src/components/ui/navigation.tsx

```diff
@@ -80,14 +81,22 @@ export function Navigation() {
             </a>
             {featureFlags?.exploreButton ? (
               <Link href="/explore" className="ml-4">
-                <div className="hover:text-foreground text-foreground/80 transition-colors">
+                <div
+                  className={clsx('hover:text-foreground text-foreground/80 transition-colors', {
```

Comment:
isnt there an active prop we can use from `Link` or am i misremembering

Reply:
rewrite in Nuxt